content/en/news/kubesphere-meetup-2020.md

```diff
@@ -0,0 +1,64 @@
+---
+
+title: 'Recap: KubeSphere & Friends 2020 Meetup'
+keywords: KubeSphere, Community, Meetup, Kubernetes
+description: KubeSphere and Friends 2020 Meetup attracted more than 100 individual community members and honored guests to share their ideas and experiences on cloud-native and Kubernetes.
+createTime: '2021-02-02'
+author: 'Feynman, Sherlock'
+
+---
+
+![Snip20210201_8](/images/news/meetup-2020/Snip20210201_8.png)
+
+Around the latest technology trend – 'cloud-native,' the annual event KubeSphere and Friends 2020, also known as Kubernetes and Cloud-Native Meetup, was held on December 19th, 2020 in Beijing, attracting more than one hundred Kubernetes enthusiasts and KubeSphere community members.
+
+Not only did our individual community members participate in the event, our guests from CNCF and AWS, and other enterprise users all shared their ideas and experiences from different perspectives.
```

Comment:
Not only did our individual community members participate in the event =>
Not only did developers from the KubeSphere team participate in the event